torch/nn/quantized/dynamic/modules/rnn.py

```diff
@@ -27,6 +29,30 @@ def __setstate__(self, state):
         self.param = torch.ops.quantized.linear_prepack(*state[0])
         self.training = state[1]
 
+    def _save_to_state_dict(self, destination, prefix, keep_vars):
+        super(PackedParameter, self)._save_to_state_dict(destination, prefix,
+                                                         keep_vars)
+        p = self.unpack()
+        destination[prefix + 'weight'] = p
+
+    def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
+                              missing_keys, unexpected_keys, error_msgs):
+        weight = state_dict[prefix + 'weight']
+        self.param = torch.ops.quantized.linear_prepack(weight)
```

Comment:
I remember `linear_prepack` takes both `weight` and `bias` as argument, did you test this change?

Reply:
You are right!